### PR TITLE
fix: show bbr error message on `celestia-appd start`

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/celestia-app/issues/3845

## Before

```
$ celestia-appd start
Error: exit status 1
```
## After

```
$ celestia-appd start

The BBR (Bottleneck Bandwidth and Round-trip propagation time) congestion control algorithm is not enabled in this system's kernel.
BBR is important for the performance of the p2p stack.

To enable BBR:
sudo modprobe tcp_bbr
net.core.default_qdisc=fq
net.ipv4.tcp_congestion_control=bbr
sudo sysctl -p

Then verify BBR is enabled:
sysctl net.ipv4.tcp_congestion_control

This node will get worse p2p performance using a different congestion control algorithm.
If you need to bypass this check use the --force-no-bbr flag.
Error: failed to execute 'sysctl net.ipv4.tcp_congestion_control' exit status 1
```

### CODE REVIEW

**Review by evan-forbes — DISMISSED** [review]
do we know why this is breaking start? if its a small issue, would it just be easier to fix that instead of revert, then re-introduce this code?

**Comment by rootulp** [comment]
idk yet

**Comment by rootulp** [comment]
I know now

```
$ celestia-appd start
Error: error while invoking 'sysctl net.ipv4.tcp_congestion_control' exit status 1
```

we should propagate this error and ideally give users an indication that they should run with the flag to skip BBR check.

**Review by staheri14 — APPROVED** [review]

**Review by evan-forbes — APPROVED** [review]
👍

**Comment by rootulp** [comment]
I want to fix golangci-lint before merging this.

File: cmd/celestia-appd/cmd/start.go [hunk 1]
```diff
@@ -570,7 +570,7 @@ func addCommands(
 func checkBBR(command *cobra.Command) error {
 	const (
 		errorMsg = `
-// The BBR congestion control algorithm does not appear to be enabled in this 
+// The BBR congestion control algorithm does not appear to be enabled in this
 // system's kernel. This is important for the p2p stack to be performant.
 //
 // to enable bbr call:
```
[the remark this answers is not in the document]
Comment (rootulp) [reply cmd/celestia-appd/cmd/start.go]: +1, we should provide instructions for as many systems as possible.

For systems that don't support BBR, we should skip checking for it. For example, IMO users on MacOS shouldn't have to pass the flag `--force-no-bbr` if it's not possible to enable BBR on MacOS.

[the remark this answers is not in the document]
Comment (rootulp) [reply cmd/celestia-appd/cmd/start.go]: https://github.com/celestiaorg/celestia-app/issues/3853

**Review by ninabarbakadze — APPROVED** [review]

**Review by staheri14 — APPROVED** [review]

**Pull request merged**